app/src/app/admin/abstractListingController.ts

```diff
@@ -73,6 +75,14 @@ namespace app.admin {
                 query._tags = {tagId:_.pluck(this.tagsToFilter, 'tagId')};
             }
 
+            if(_.isEmpty(query)) {
+                return this.entitiesPaginator.reset().getPage(1)
+                    .then((entities) => {
+                        this.entities = entities;
```

Comment:
This block is partially duplicate now, and will fail if there are no results rather than return empty array. You should be able to just return the promise to a variable then have one `then` block (same as what is already returned)